pkg/apis/kops/v1alpha1/instancegroup.go

```diff
@@ -96,6 +96,8 @@ type InstanceGroupSpec struct {
 	Zones []string `json:"zones,omitempty"`
 	// SuspendProcesses disables the listed Scaling Policies
 	SuspendProcesses []string `json:"suspendProcesses,omitempty"`
+	// ExternalLoadBalancers define loadbalancers that should be attached to the instancegroup
+	ExternalLoadBalancers []LoadBalancer `json:"externalLoadBalancers,omitempty"`
```

Comment:
Do you have an example of what you'd add? I can't think of anything you'd set as a whole as opposed to on an individual load balancer-level.

Reply:
Not off the top of my head, I guess I envisioned the  LoadBalancer Struc to be more of a catch all for anything LB related which is why I was thinking of leaving it open ended. But since I can't think of anything right now and you are doing the work I'll leave it up to you. 😄

Reply:
The closest related setting I can come up with is the ASG setting to use EC2 or ELB health checks. But that'd apply to all loadbalancers including the one that Kops creates for the masters if the cluster spec has `spec.api.loadBalancer` set.

Also using the ELB health check in general feels a bit scary to me for k8s, as a single pod's failing healthcheck could potentially cause autoscaling to bring down the node attached to the load balancer even if that pod was running on another node.